### ISSUE TITLE
Pinned routes hidden on short screen

### ISSUE THREAD

**Issue by midnightcomm** [mekto/brouter-online#1]
When the number of pinned routes exceed the available vertical window size, the bottom ones are hidden. For quite smaller screens this would get really annoying. Work around is to just not have as many routes pinned at once.

Perhaps some sort of scrolling is needed.

![screenshot from 2016-02-08 12 52 29](https://cloud.githubusercontent.com/assets/5687236/12878177/412858fc-ce63-11e5-915c-1bfb1d0c02fc.png)

**Comment by mekto** [comment]
Yes, you're right, current UI is not very usable on small screens especially with pinned routes. Possible solutions I can think of right now are:
1. scrolling (as you suggested)
2. collapsible route cards
3. stacked cards and only one fully visible on click or hover
   or maybe mixture of the above features.

I will try to address this issue in next release.
